RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+1.25.0
+------
+* Improve icon on the "Take a Video" media option
```

Comment:
This change affected Android only, since on iOS we are not showing images on Media Options.
Please let's prefix these cases with `[Android]` or `[iOS]` as needed 👍

Reply:
Sorry I didn't see this earlier. @SergioEstevao is in the process of updating this as part of the 1.25. release process.